templates/blog.html

```diff
@@ -15,7 +15,7 @@ <h2>{{ this.title }}</h2>
       <p>{{ this.description }}</p>
     </header>
     <center>  
-      <a href="{{ this.link|url }}" class="button primary fit" >{{ this.spruch }}
+      <a href="{{ this.link|url }}" class="button primary" >{{ this.spruch }}
```

Comment:
@DO1JLR hast du spontan eine Ahnung was hier der primary tag macht? Ich kann den nicht wirklich zuordnen

Reply:
primary - oder special (bin mir nicht genau sicher was in welchen template war) hebt die buttons nochmal optisch hervor.

normal ohne special oder primary:
![grafik](https://user-images.githubusercontent.com/16953664/60376319-5f43a880-9a0e-11e9-8431-31feee651fda.png)

button mit special oder primary:
![grafik](https://user-images.githubusercontent.com/16953664/60376330-8306ee80-9a0e-11e9-8343-29a853b3acdd.png)


Ich vermute mal, da ist wohl so nen "Ehl Drai Deh" beim schreiben des template durcheinander gekommen und hat primary und special verwechselt.

Reply:
das ist special primary gibt es in der button scss gar nicht